src/onefetch/cli_utils.rs

```diff
@@ -0,0 +1,94 @@
+use crate::onefetch::{ascii_art::AsciiArt, error::*, info::Info, language::Language};
+use colored::Color;
+use std::env;
+use std::io::Write;
+use strum::IntoEnumIterator;
+
+pub struct Printer<W> {
+    writer: W,
+    info: Info,
+}
+
+impl<W: Write> Printer<W> {
+    pub fn new(writer: W, info: Info) -> Self {
+        Self { writer, info }
+    }
+
+    pub fn print(&mut self) -> Result<()> {
+        let center_pad = "   ";
+        let info_str = format!("{}", &self.info);
+        let mut info_lines = info_str.lines();
+        let colors: Vec<Color> = Vec::new();
+        let mut buf = String::new();
+        if let Some(custom_image) = &self.info.config.image {
+            if let Some(image_backend) = &self.info.config.image_backend {
+                buf.push_str(&image_backend.add_image(
+                    info_lines.map(|s| format!("{}{}", center_pad, s)).collect(),
+                    custom_image,
+                ));
+            } else {
+                panic!("No image backend found")
```

Comment:
Since `Printer::print` returns a `Result`, it might make sense for this `panic!` to be replaced with an `Err`.

Reply:
I actually managed to remove this else stmt altogether in #289